ingredient/ingredient-key-vault/README.md

```diff
@@ -0,0 +1,67 @@
+## Changelogs
+
+* [@azbake/ingredient-key-vault]
+
+## Overview
+
+The Key Vault ingredient is a plugin for bake. When included in a recipe, this will create an [Key Vault](https://azure.microsoft.com/en-us/services/key-vault/).
+
+## Usage
+
+```yaml
+name: My package
+shortName: mypkg
+version: 0.0.1
+ingredients:
+  - "@azbake/ingredient-key-vault@~0"
+resourceGroup: true
+recipe:
+  keyvault:
+    properties:
+      type: "@azbake/ingredient-key-vault"
+      parameters:
+        name: "[keyvaultutils.create_resource_name()]"       
+        sku: "standard"                      
+        networkAcls:
+          defaultAction: "Deny"
+          bypass: "AzureServices"
+          ipRules: []
+          virtualNetworkRules: []
+```
+
+| property | required | description |
+| -------- | -------- | ----------- |
+| name | yes | The name for the Key Vault |
+| location | no | Sets the location. Defaults to the parent resource group location. |
+| sku | yes | SKU for the vault |
+| accessPolicies | yes | The access policies defined for this vault |
+| tenant | no | Specifies the Azure Active Directory tenant ID that should be used for authenticating requests to the key vault |
+| enabledForDeployment | no | Boolean flag to specify whether Azure Virtual Machines are permitted to retrieve certificates stored as secrets from the key vault |
+| enabledForTemplateDeployment | no | Boolean flag to specify whether Azure Disk Encryption is permitted to retrieve secrets from the vault and unwrap keys |
```

Comment:
I think the description for `enabledForTemplateDeployment` and `enabledForDiskEncryption` are switched

Reply:
Updated the description.